worldengine/draw.py

```diff
@@ -307,7 +307,7 @@ def get_biome_color_based_on_elevation(world, elev, x, y):
     modification_amount = int(elev / BASE_ELEVATION_INTENSITY_MODIFIER)
     base_elevation_modifier = (modification_amount, modification_amount, modification_amount)
 
-    this_tile_color = add_colors(biome_color, noise, base_elevation_modifier)
+    this_tile_color = add_colors(biome_color, noise, base_elevation_modifier)  # this can lead to an overflow, i.e. a value above 255
```

Comment:
I think this might not be perfect yet. Maybe an overflow wouldn't even cause problems, I am not sure.